.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
```

Comment:
Think a mention of the CVE/Security stuff pointing to the SECURITY.md would be good. There is a small link on the new issues, but it might help to pull up here too with a version of the content from https://github.com/nodejs/docker-node/security/policy

Reply:
Addressing in the next commit.